lib/config/config.go

```diff
@@ -52,24 +51,25 @@ func IsFeatureBranch(branchName string) bool {
 
 func IsPerennialBranch(branchName string) bool {
   perennialBranches := GetPerennialBranches()
-  return sort.SearchStrings(perennialBranches, branchName) < len(perennialBranches)
```

Comment:
[SearchStrings](https://golang.org/pkg/sort/#SearchStrings) tells you where to insert the value into the sorted array and doesn't tell us if the string appears in the list. Haven't run into in any feature tests yet as we I don't know if we have any with both perennial branches and feature branches